### DISCUSSION TITLE
Welcome to nussl Discussions!

### DISCUSSION THREAD

**Discussion by ethman** [nussl/nussl#194] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ethman** [comment]
Hello!

I'm Ethan. I'm a PhD Candidate at Northwestern University in the Interactive Audio Lab led by Prof. Bryan Pardo. My research focuses on source separation, automatic music transcription, and open source datasets and applications. I'm a guitar player by training and these days I play a lot of solo guitar (I'm really into [American Primitive](https://en.wikipedia.org/wiki/American_primitive_guitar)).

I'm one of the original creators of `nussl`. We actively use nussl for experimenting and prototyping new source separation algorithms and ideas, some of which make it into the library. We're working on filling out the functionality, squashing bugs, and overall making nussl an easy, go-to solution for all things source separation.

I look forward to hearing from everyone!

**Comment by abugler** [comment]
Hi!

I'm Andreas, and I am a Undergraduate Student at Northwestern University studying computer science, and I also work in the Interactive Audio Lab.  Recently, I have been working on projects involving audio source separation with @ethman and @pseeth. 

Besides being an avid user of nussl, I have been contributing to nussl since `1.1.1`!
